tests/integration_tests/access_tests.py

```diff
@@ -103,11 +103,7 @@ def tearDownClass(cls):
             db.session.delete(security_manager.find_role(SCHEMA_ACCESS_ROLE))
             db.session.commit()
 
-    def setUp(self):
-        self.login("admin")
-
     def tearDown(self):
-        self.logout()
```

Comment:
The logout is unnecessary as the client only persists for the lifetime of the test.